Modules/Loadable/Segmentations/Logic/vtkSlicerSegmentationsModuleLogic.cxx

```diff
@@ -1441,6 +1441,34 @@ bool vtkSlicerSegmentationsModuleLogic::ImportLabelmapToSegmentationNode(vtkMRML
     padder->Update();
     labelOrientedImageData->ShallowCopy(padder->GetOutput());
 
+    int ret = vtkOrientedImageDataResample::IsSegmentationScalarTypeValid(labelOrientedImageData);
+    switch (ret)
+    {
+      case vtkOrientedImageDataResample::TYPE_CONVERSION_TRUNCATION_NEEDED:
+        vtkGenericWarningMacro(
```

Comment:
It would be nice to able to return this as userMessages to allow the error message to be displayed in the GUI.